library/src/scripts/forms/select/SelectBox.tsx

```diff
@@ -84,21 +84,23 @@ export default class SelectBox extends React.Component<ISelfLabelledProps | IExt
 
     public render() {
         const classes = selectBoxClasses();
+
         const classesDropDown = dropDownClasses();
-        const classesMetas = metasClasses();
         const selectItems = this.props.children.map((child, i) => {
             const selected = this.state.selectedIndex === i;
             return (
                 <DropDownItemButton
                     key={this.props.id + "-item" + i}
                     className={classNames({ isSelected: child.selected })}
                     name={child.name}
-                    onClick={this.handleClick.bind(this, child, i)}
+                    onClick={() => {
```

Comment:
Use a real link here. No need to use a button. It's not semantic and it's not good for accessibility. We have a "DropDownItemLink" component